components/02-form-elements/mutually-exclusive/mutually-exclusive.js

```diff
@@ -0,0 +1,37 @@
+import domready from '../../../assets/js/domready';
+
+const exclusiveCheckboxWrapperClass = 'field--exclusive'
+const exclusiveCheckboxGroupClass = 'js-exclusive-checkbox-group';
+const exclusiveCheckboxClass = 'js-exclusive-checkbox';
+const exclusiveCheckboxAlertClass = 'js-exclusive-checkbox-alert';
+
+export default function mutuallyExclusiveCheckboxes() {
+  const exclusiveCheckboxWrapperElements = document.getElementsByClassName(exclusiveCheckboxWrapperClass);
+  for (let exclusiveCheckboxWrapperElement of exclusiveCheckboxWrapperElements) {
+    let exclusiveCheckboxGroupElements = exclusiveCheckboxWrapperElement.getElementsByClassName(exclusiveCheckboxGroupClass);
+    let exclusiveCheckboxElement = exclusiveCheckboxWrapperElement.getElementsByClassName(exclusiveCheckboxClass);
+    let exclusiveCheckboxAlert = exclusiveCheckboxWrapperElement.getElementsByClassName(exclusiveCheckboxAlertClass);
+    for (let exclusiveCheckboxGroupElement of exclusiveCheckboxGroupElements) {
+      exclusiveCheckboxGroupElement.onclick = function() {
+        exclusiveCheckboxAlert[0].innerHTML = '';
```

Comment:
I would store a reference to this `exclusiveCheckboxAlert[0]` instead of grabbing the first one from an array every time.